ixwebsocket/IXHttpClient.cpp

```diff
@@ -406,10 +406,10 @@ namespace ix
             ss << headers["Content-Length"];
             ss >> contentLength;
```

Comment:
Can you keep the payload.reserve call ? It will be better in the general case.

Reply:
Hey! I moved the `reserve` call a couple of lines down (line 429, after the `readBytes` call), because it shouldn't be called for the case where a chunk callback is used. Is that good enough, or do you want it to happen before `readBytes`?

Reply:
You are right that it does not matter, it just has to be done right before we write into the buffer, so the place where you moved it is fine.